tests/controls/slidertest.cpp

```diff
@@ -188,21 +188,28 @@ void SliderTestCase::Range()
 
 void SliderTestCase::Thumb()
 {
-#if wxUSE_UIACTIONSIMULATOR && !defined(__WXGTK__)
+#if wxUSE_UIACTIONSIMULATOR
     EventCounter track(m_slider, wxEVT_SCROLL_THUMBTRACK);
     EventCounter release(m_slider, wxEVT_SCROLL_THUMBRELEASE);
     EventCounter changed(m_slider, wxEVT_SCROLL_CHANGED);
 
+#ifdef __WXGTK__
+    // Give m_slider a reasonable width.
+    m_slider->SetSize(100, -1);
+#endif // __WXGTK__
+
```

Comment:
It should be fine to remove this when/if #2012 is merged.

Reply:
Update: #2012 _is_ merged now.

Reply:
Done!